### PR TITLE
Airport challenge

### PR DESCRIPTION
**Pull request by kriszv2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+## 2. In terminal type node specRunner.js
+
+## 3. run multiple tests because of test number 7 and 8 (weather landing check)
+
+## 4. With the exception of test number 7 and 8, all tests should pass on every run
+
+# Airport Challenge
+
+``````
          ______
```
Comment (kriszv2) [inline README.md:22]: For some reason my airport.js file starts with a small a, even though the same files on my pc start with capital letter.

File: test/airport.spec.js [hunk 2]
```diff
@@ -0,0 +1,288 @@
+const { assertEquals, assertTrue, assertFalse } = require(`../framework`);
+const Airport = require(`../src/Airport`);
```
Comment (kriszv2) [inline test/airport.spec.js:2]: Have written that in the README as well, but for some reason the airport.js file changed from capital A to small a, therefore this require will not work properly, on my PC it is still capital A, don't know why.

**Review by kriszv2 — COMMENTED** [review]
Only found a naming problem, which I don't know how could have occured, my airport.js file starts with a small "a", even though on my pc it is a capital A, and even in the file itself the import is capital A.

File: test/airport.spec.js (hunk 2 above)
Comment (EdWright75) [reply test/airport.spec.js:2]: Windows systems are case insensitive when it comes to file names - unix-based systems (Linux and MacOS) are generally case-sensitive.  Not sure why this has happened - I suspect Windows is doing too much!

**Pull request closed without merging**